x/clp/keeper/calculations_test.go

```diff
@@ -438,56 +438,56 @@ func TestKeeper_CalculateWithdrawal(t *testing.T) {
 		asymmetry            sdk.Int
 		panicErr             string
 	}{
-		{
-			name:                 "fail to convert nativeAssetBalance to Dec",
-			poolUnits:            sdk.NewUint(1),
-			nativeAssetBalance:   "10000000000000000000000000000000000000000000000000000000000000000000000000",
-			externalAssetBalance: "1",
-			lpUnits:              "1",
-			wBasisPoints:         "1",
-			asymmetry:            sdk.NewInt(1),
-			panicErr:             "fail to convert 10000000000000000000000000000000000000000000000000000000000000000000000000 to cosmos.Dec: decimal out of range; bitLen: got 303, max 256",
-		},
-		{
-			name:                 "fail to convert externalAssetBalance to Dec",
-			poolUnits:            sdk.NewUint(1),
-			nativeAssetBalance:   "1",
-			externalAssetBalance: "10000000000000000000000000000000000000000000000000000000000000000000000000",
-			lpUnits:              "1",
-			wBasisPoints:         "1",
-			asymmetry:            sdk.NewInt(1),
-			panicErr:             "fail to convert 10000000000000000000000000000000000000000000000000000000000000000000000000 to cosmos.Dec: decimal out of range; bitLen: got 303, max 256",
-		},
-		{
-			name:                 "fail to convert lpUnits to Dec",
-			poolUnits:            sdk.NewUint(1),
-			nativeAssetBalance:   "1",
-			externalAssetBalance: "1",
-			lpUnits:              "10000000000000000000000000000000000000000000000000000000000000000000000000",
-			wBasisPoints:         "1",
-			asymmetry:            sdk.NewInt(1),
-			panicErr:             "fail to convert 10000000000000000000000000000000000000000000000000000000000000000000000000 to cosmos.Dec: decimal out of range; bitLen: got 303, max 256",
-		},
-		{
-			name:                 "fail to convert wBasisPoints to Dec",
-			poolUnits:            sdk.NewUint(1),
-			nativeAssetBalance:   "1",
-			externalAssetBalance: "1",
-			lpUnits:              "1",
-			wBasisPoints:         "10000000000000000000000000000000000000000000000000000000000000000000000000",
-			asymmetry:            sdk.NewInt(1),
-			panicErr:             "fail to convert 10000000000000000000000000000000000000000000000000000000000000000000000000 to cosmos.Dec: decimal out of range; bitLen: got 303, max 256",
-		},
-		{
-			name:                 "fail to convert asymmetry to Dec",
-			poolUnits:            sdk.NewUint(1),
-			nativeAssetBalance:   "1",
-			externalAssetBalance: "1",
-			lpUnits:              "1",
-			wBasisPoints:         "1",
-			asymmetry:            sdk.Int(sdk.NewUintFromString("10000000000000000000000000000000000000000000000000000000000000000000000000")),
-			panicErr:             "fail to convert 10000000000000000000000000000000000000000000000000000000000000000000000000 to cosmos.Dec: decimal out of range; bitLen: got 303, max 256",
-		},
+		//{
```

Comment:
Can you give a reason in the comments for commenting out these tests?

Reply:
@Jedi2002 I would recommend keeping the test case and increasing the number of digits of the asymmetry decimal to make it panic the same way you do with the other use cases.

Reply:
The only one that's commented not is for asymmetry which uses SDK.INT . 
The panic comes from creating the UINT which is part of the test setup, not the function it's trying to test.

if the goal is to test sdk.NewUintFromString , please let me know , we can have a separate test case for that specifically

Reply:
@Jedi2002 so are you saying those test cases you commented are not relevant anymore and if so can you explain why, thanks

Reply:
Hey, @canercandan  I think these comments are on an older commit, all other tests except the one for asymmetry. 
The asymmetry test panics at test data generation and not the function it's trying to test. We can chant about it over a call if you need more clarity.